gf-guide/metrics.md

```diff
@@ -151,22 +151,22 @@ Expected result: vertical metrics should be around 130% of UPM. Anything greater
 A new Latin family has the following qualities:
 
 -   UPM is `1000`
--   `yMax` of tallest `A breve acute` in the familly (black for this example) = `1015`
--   `yMin` of deepest a-z letter (`g` bold in this family) = `210`
+-   `Abreveacute U+1EAE` glyph `yMax` value (taller in the family, Black font for this example) = `1015`
+-   Deepest a-z glyph in the family `yMin` value (`g` bold in this example) = `-210`
 -   Caps height (`H`or `Z` bbox height) = `700`
--   Family's `yMax = 1116` (Black `A breve hookabove` for this family)
```

Comment:
Black -> max Weight ?

Reply:
Black being the max weight of this hypothetical family.

Reply:
okay cool

Reply:
Done, in line 154.